localization/en.ftl

```diff
@@ -250,6 +250,15 @@ intellisense-opt-subscriptify-name = Auto-Subscriptify
 intellisense-opt-subscriptify-desc = Automatically converts text of variables/functions with subscripts when they are typed without subscripts.
 intellisense-jump2def-menu-instructions = has multiple definitions. Pick one from below to jump to.
 
+## My Expressions Library
+my-expressions-library-name = My Expressions Library
+my-expressions-library-desc = Create and use a reusable library of expressions in multiple graphs.
+my-expressions-library-opt-libraryGraphHashes-name = Graph Hashes
+my-expressions-library-opt-libraryGraphHashes-desc = A list of graph hashes (those 10-digit sequences of letters and numbers that uniquely identify a graph) from which to source expressions. Each hash should be on its own line.
+my-expressions-library-pillbox-menu = My Expressions Library
+my-expressions-library-did-not-load = My Expressions Library: The graphs with the following hashes did not load: { $hashes }. 
+my-expressions-library-empty-library = Your library is currently empty. Add a graph to it in the DesModder menu under "My Expressions Library" in the "Utility" section in order to quickly have access to that graph's expressions elsewhere. 
```

Comment:
We could provide a way to jump to that part of the settings menu, instead of giving directions.

Reply:
Actually, instead of that, make it so you can add to the library in the same spot where you can read/insert from the library. Also makes it clear which graph hash corresponds to which graph.

Reply:
Good idea! I'll add that in.